docs/upcoming_changes/9034.cuda.rst

```diff
@@ -0,0 +1,6 @@
+Report absolute path of libcuda.so on Linux
+===========================================
+
+`numba -s` now reported the absolute path to libcuda.so on Linux, to aid
```

Comment:
```suggestion
``numba -s`` now reports the absolute path to ``libcuda.so`` on Linux, to aid
```
think RST needs double backticks for code statements? (and line wrap)

Reply:
Ah, I forgot this was RST! (despite it being in the name)